documentation/ipi-install/modules/ipi-install-bmc-addressing.adoc

```diff
@@ -0,0 +1,144 @@
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish://<out-of-band-ip>/redfish/v1/Systems/1
+          username: <user>
+          password: <password>
+          disableCertificateVerification: True
+----
+
+
+.RedFish for Dell
+
+To enable RedFish, use `redfish://` or `redfish+http://` to disable TLS. The installer requires both the hostname or the IP address and the path to the system ID. The following example demonstrates a RedFish configuration within the `install-config.yaml` file.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish://<out-of-band-ip>/redfish/v1/Systems/System.Embedded.1
+          username: <user>
+          password: <password>
+----
+
+While it is recommended to have a certificate of authority for the
+out-of-band management addresses, you must include `disableCertificateVerification: True` in the `bmc` configuration if using self-signed certificates. The following example demonstrates a RedFish configuration using the `disableCertificateVerification: True` configuration parameter within the `install-config.yaml` file.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish://<out-of-band-ip>/redfish/v1/Systems/System.Embedded.1
+          username: <user>
+          password: <password>
+          disableCertificateVerification: True
+----
+
+[NOTE]
+====
+Currently RedFish is only supported on Dell with iDRAC firmware version `4.20.20.20` or higher for IPI on Bare metal deployments.
+====
+
+
+.RedFish Virtual Media for HPE
+
+To enable RedFish Virtual Media for HPE servers, use `redfish-virtualmedia://` in the `address` setting. The following example demonstrates using RedFish Virtual Media within the `install-config.yaml` file.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish-virtualmedia://<out-of-band-ip>/redfish/v1/Systems/1
+          username: <user>
+          password: <password>
+----
+
+
+.RedFish Virtual Media for Dell
+
+For RedFish Virtual Media on Dell servers, use `idrac-virtualmedia://` in the `address` setting. The following example demonstrates using iDRAC Virtual Media within the  `install-config.yaml` file.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: idrac-virtualmedia://<out-of-band-ip>/redfish/v1/Systems/System.Embedded.1
+          username: <user>
+          password: <password>
+----
+
+
+[NOTE]
+====
+`idrac-virtualmedia` requires iDRAC firmware version 4.20.20.20 or higher.
```

Comment:
need another note about available in version 4.6.1 or higher -- see email

Reply:
Currently RedFish is only supported on Dell with iDRAC firmware version `4.20.20.20` or higher for IPI on Bare metal deployments and from {product-title} version greater or equal to `4.6.1`.